docs/devel/e2e-tests.md

```diff
@@ -102,6 +102,18 @@ As mentioned earlier there are a host of other options that are available, but a
 - `rm -rf /var/run/kubernetes`, clear kube generated credentials, sometimes stale permissions can cause problems.
 - `sudo iptables -F`, clear ip tables rules left by the kube-proxy.
 
+## Kinds of tests
+
+We are working on implementing clearer partitioning of our e2e tests to make running a known set of tests easier (#10548).  Tests can be labeled with any of the following labels, in order of increasing precedence (that is, each label listed below supersedes the previous ones):
+
+- If a test has no labels, it is expected to run fast (under five minutes), be able to be run in parallel, and be consistent.
```

Comment:
1-2 minutes has been considered the maximum up to now.  You currently have around 150 tests which you run in serial, and want them to complete in around 1 hour.  That give's an average of around 24 seconds per test.  5 minutes seems way to high to achieve those goals.

http://kubekins.dls.corp.google.com/job/kubernetes-e2e-gce/lastCompletedBuild/testReport/history/